cmake/targets/common.cmake

```diff
@@ -87,7 +87,7 @@ set_source_files_properties("${CMAKE_SOURCE_DIR}/src/upnp.cpp"
 # third-party/nanors
 set_source_files_properties("${CMAKE_SOURCE_DIR}/third-party/nanors/rs.c"
         DIRECTORY "${CMAKE_SOURCE_DIR}" "${TEST_DIR}"
-        PROPERTIES COMPILE_FLAGS "-include deps/obl/autoshim.h -ftree-vectorize")
+        PROPERTIES COMPILE_FLAGS "-include deps/obl/autoshim.h -ftree-vectorize -mssse3 -funroll-loops")
```

Comment:
We should double-check that passing `-mssse3` won't break non-x86 builds (or only do it on x86).

@ReenigneArcher do we have an non-x86 builds in pre-merge CI?

Reply:
All arm builds are after merge due to the time required with qemu. I can build them locally though if needed.

Reply:
I'm having issues building on the ubuntu 22.04 and debian image, but the failure is not related to this as I cannot even dependencies.

```txt
Processing triggers for libc-bin (2.35-0ubuntu3.8) ...
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
Segmentation fault
qemu: uncaught target signal 11 (Segmentation fault) - core dumped
Segmentation fault
dpkg: error processing package libc-bin (--configure):
installed libc-bin package post-installation script subprocess returned error exit status 139
Processing triggers for ca-certificates (20230311ubuntu0.22.04.1) ...
Updating certificates in /etc/ssl/certs...
0 added, 0 removed; done.
Running hooks in /etc/ca-certificates/update.d...
done.
Errors were encountered while processing:
libc-bin
E:Sub-process /usr/bin/dpkg returned an error code (1)
```

I'm not sure what's going on with that.

Reply:
@cgutman Sounds good. SSSE3 change has its own commit https://github.com/LizardByte/Sunshine/pull/2803/commits/374485e244e2cee39494ce0dbc6d102aeb862fc2, so just exclude it during rebase. I can force-push it myself if you prefer that, but I rather dislike doing it after code review has already been done.

Reply:
OK, I'll drop that commit, squash the fixups, and get this merged. Thanks for tackling this one.